missing-types.d.ts

```diff
@@ -67,17 +66,6 @@ interface ResultMeta {
   }>;
 }
 
-interface ToolSummary {
-  tool: BuildTool;
-  total: number;
-  possible: number;
-}
-
-interface Link {
-  title: string;
-  href: string;
-}
```

Comment:
These definitions were only used in one place each, so I moved them alongside their usage.